.gitignore

```
@@ -0,0 +1,2 @@
+cq-source-onfleet
+.vscode
```

```suggestion
.vscode
dist
```

Otherwise the release workflow will fail.

See more in https://github.com/cloudquery/cloudquery/pull/7608